packages/graphback/src/GraphQLBackendCreator.ts

```diff
@@ -88,27 +105,73 @@ export class GraphQLBackendCreator {
   }
 
   public async createClient(): Promise<Client> {
-    const clientGenerator = new ClientGenerator(this.inputContext)
+    const clientGenerator = new ClientGenerator(this.inputContext);
 
-    return clientGenerator.generate()
+    return clientGenerator.generate();
   }
 
+  public async migrateDatabase(
+    initializationStrategy: DatabaseInitializationStrategy
+  ): Promise<void> {
+    const context = this.inputContext.filter(
```

Comment:
Would it be possible to have separate class in migration folder.
Sorry for being anal on API/BreakingChage/Responsibility rules

`GraphQLBackendCreator` is kinda front door to our library so I would keep this clean. 
Ideally here we should instantiate things that do migration.